content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -235,15 +235,21 @@ spec:
 If this feature is disabled, the `adminAccess` field will be removed
 automatically when creating such a ResourceClaim.
 
-Admin access is a privileged mode which should not be made available to normal
-users in a multi-tenant cluster. Cluster administrators can restrict usage of
+Admin access is a privileged mode and should not be granted to regular users in
+multi-tenant clusters. Starting with Kubernetes v1.33, only users authorized to
+create ResourceClaim or ResourceClaimTemplate objects in namespaces labeled with
+`resource.k8s.io/admin-access: "true"` (case-sensitive) can use the
+`adminAccess` field. This ensures that non-admin users cannot misuse the
+feature. 
+
+Prior to Kubernetes v1.33, cluster administrators can restrict usage of
 this feature by installing a validating admission policy similar to the following
 example. Cluster administrators need to adapt at least the names and replace
 "dra.example.com".
 
 ```yaml
 # Permission to use admin access is granted only in namespaces which have the
-# "admin-access.dra.example.com" label. Other ways of making that decision are
+# "resource.k8s.io/admin-access" label. Other ways of making that decision are
```

Comment:
Was this `admin-access.dra.example.com` label documented incorrectly for pre-1.33? Or did the label change in the 1.33 release cycle?

Reply:
The "admin-access.dra.example.com" was correct for this example VAP. Using "resource.k8s.io/admin-access" here is (strictly speaking) not correct. Admins should use a name that they control, whereas "k8s.io" is reserved for Kubernetes usage.

We could argue that this label name could be used on Kubernetes < 1.33 to emulate the behavior of Kubernetes >= 1.33. But that then needs to be called out more explicitly.

Reply:
I can revert this change if you prefer. Wanted to make the label consistent to avoid confusion for users reading this for the first time. would it also make sense to remove the VAP example and focus on 1.33+?

Reply:
That might be best. We typically don't document something for older releases in the documentation of the current release.

Reply:
> Admins should use a name that they control, whereas "k8s.io" is reserved for Kubernetes usage.

I'm not sure it's wrong to use the official label once we register it. An administrator can declare "if a namespace has this (registered, official) label, here's my opinion on whether changes relating to that namespace can be admitted".

On _main_ we haven't registered the label yet. We can, any time we want.

Reply:
I think the v1.32 documentation should recommend using `resource.k8s.io/admin-access` but only once https://kubernetes.io/docs/reference/labels-annotations-taints/ includes a mention of that label key.

Reply:
removed vap from 1.33 doc. PTAL